Tasks/UseDotNetV2/task.json

```diff
@@ -13,8 +13,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 2,
-        "Minor": 198,
-        "Patch": 1
+        "Minor": 200,
```

Comment:
here and in all such places. Also consistency issue ()Here its 200 in other places 199)

Reply:
fixed with 204